### PR TITLE
text-splitters[minor], langchain[minor], community[patch], templates, docs: langchain-text-splitters 0.0.1

### PR DESCRIPTION
**Pull request by baskaryan**

### CODE REVIEW

File: libs/text-splitters/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  "E",  # pycodestyle
+  "F",  # pyflakes
+  "I",  # isort
+  "T201", # print
+]
+
+[tool.mypy]
+disallow_untyped_defs = "True"
+
+[[tool.mypy.overrides]]
```
Comment (baskaryan) [inline libs/text-splitters/pyproject.toml:73]: @efriis pretty nifty, lets you skip import errors for specific packages that don't have stubs

File: libs/partners/airbyte/langchain_airbyte/document_loaders.py
Comment (efriis) [inline libs/partners/airbyte/langchain_airbyte/document_loaders.py:1]: update lint dep to be langchain-text-splitters instead of langchain

**Review by efriis — COMMENTED** [review]
Guessing you're working on these already but probs need to sort out some of the re-exports and dependencies

**Comment by efriis** [comment]
Oh also should we 0.1.0 for poetry version bump benefits like partner packages?

**Pull request merged**